backend/capellacollab/projects/routes.py

```diff
@@ -140,8 +140,8 @@ def delete_project(
     project: DatabaseProject = Depends(get_existing_project),
     db: Session = Depends(get_db),
 ):
-    crud.delete_project(db, project)
-    return None
+    if not project.models:
+        crud.delete_project(db, project)
```

Comment:
```suggestion
    if not (project.models or project.users):
        crud.delete_project(db, project)
```
Without this check, we get an error when we delete a project that has users assigned to it. It should then also not be possible to click on the button in the frontend (as you did with the models).